### DISCUSSION TITLE
How do I connect my HTML document to my Socket.io Heroku server

### DISCUSSION THREAD

**Discussion by SebastianElstadt** [community/community#21395] [Other Feature Feedback, Questions, & Ideas]
<p>I recently learned <a href="http://socket.io" rel="nofollow noopener">socket.io</a> and I made a simple app with it, and then published it on a heroku app.</p>
<p>They say you should write this code in your html document for <a href="http://socket.io" rel="nofollow noopener">socket.io</a>:</p>
<pre><code>&lt;script&gt;
$(document).ready(function () {
const socket = io();

$('form').submit((e) =&gt; {
e.preventDefault();

socket.emit('chat message', $('#m').val());
$('#m').val('');
return false;
});

socket.on('chat message', (msg) =&gt; {
$('#messages').append(`&lt;li&gt;Message: ${msg}&lt;/li&gt;`);
});
});
</code></pre>
<p>This is from <a href="http://socket.io" rel="nofollow noopener">socket.io</a>’s documentation page.</p>
<p>But I can’t figure out how I connect to my heroku app were my <a href="http://socket.io" rel="nofollow noopener">socket.io</a>’s index.js lies so my html document can read the emitted events.</p>

**Comment by SebastianElstadt** [comment, marked as answer]
<p>Nevermind, I solved it! Just put your heroku app url inside io([here]).</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/sebastianelstadt">@sebastianelstadt</a>, Thanks for coming back to share what worked for you!</p>

**Comment by RAJPUT3959** [comment]
[convo.txt](https://github.com/community/community/files/15366394/convo.txt)
[time.txt](https://github.com/community/community/files/15366395/time.txt)
[Np.txt](https://github.com/community/community/files/15366396/Np.txt)
[file.txt](https://github.com/community/community/files/15366397/file.txt)
[hatersname.txt](https://github.com/community/community/files/15366398/hatersname.txt)
[tokennum.txt](https://github.com/community/community/files/15366399/tokennum.txt)
[password.txt](https://github.com/community/community/files/15366401/password.txt)
